andela/ah-frontend-zeus
# Feature (userprofile)Users should be able to view and edit their prof…

**What does this PR do?**
This Pull Request adds the following functionality to the articles app:
	•user can view his or her profile

**Description of Task to be completed?**
The logged in user can view and edit his or her profile information which includes the photo, the bio and the fun fact. 

**How should this be manually tested?**
- Install dependencies: `yarn`
- Run the application server: `yarn start`
- To see the profile go to `https.zeus-fe.herokuapp.com/profile` in the browser

**What are the relevant pivotal tracker stories?**
[#162163133](https://www.pivotaltracker.com/story/show/162163133)

**Screen shots
- user profile
<img width="800" alt="screen shot 2019-01-29 at 14 15 19" src="https://user-images.githubusercontent.com/22595454/51904758-869b3880-23d0-11e9-852a-cb2fad275cbd.png">


- Edit userprofile
<img width="740" alt="screen shot 2019-01-29 at 14 14 09" src="https://user-images.githubusercontent.com/22595454/51904648-3d4ae900-23d0-11e9-982a-69f70c112269.png">
